packages/api/src/controllers/stripe.ts

```diff
@@ -67,6 +78,16 @@ export const reportUsage = async (req, adminToken) => {
 };
 
 async function reportUsageForUser(req, user, adminToken) {
```

Comment:
Can you add types here? All these should already exist so no reason to use `any` here.